### ISSUE TITLE
Expose sync-status of device-groups via bigip_device_info

### ISSUE THREAD

**Issue by simonkowallik** [F5Networks/f5-ansible#1778]
##### ISSUE TYPE
 - Feature Idea

##### COMPONENT NAME
`bigip_device_info`

##### ANSIBLE VERSION
```
Ansible 2.9.7
f5-ansible collection 1.4.0
```

##### PYTHON VERSION
```
3.6.9
```

##### BIGIP VERSION
```
Any supported
```

##### SUMMARY
The Configuration Synchronisation Status of Device Groups on BIG-IP is often an important factor to decide whether to modify or apply new configurations. In particular in HA pairs the sync status of the sync-failover device-group is an important state information to determine the state/readiness of the Standby device to take over traffic processing.

It would be very helpful to have the sync status of each device group available through `bigip_device_info`.

##### EXPECTED RESULTS
Provide device group sync status.

For example as an additional field in `device-groups`:
```yaml
     bigip_device_info:
        gather_subset:
          - device-groups
```
Or via a seperate `sync-status` subset:
```yaml
     bigip_device_info:
        gather_subset:
          - sync-status
```

##### ACTUAL RESULTS
Currently the sync status is only available through a workaround, in addition the output is hard to parse:

```yaml
     - name: get device-group sync-status
        bigip_command:
          provider: "{{ provider }}"
          commands:
            - show cm sync-status field-fmt
```
Regarding this workaround also note https://github.com/F5Networks/f5-ansible/issues/1734

**Comment by focrensh** [comment]
Tracking with FMFA-551
